keras_cv/models/backbones/vit/vit_backbone_presets.py

```diff
@@ -0,0 +1,229 @@
+# Copyright 2023 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""ViT model preset configurations."""
+
+backbone_presets_no_weights = {
+    "vit18": {
+        "metadata": {
+            "description": (
+                "ViT model with 18 layers where the batch normalization "
+                "and ReLU activation are applied after the convolution layers "
+                "(v1 style)."
+            ),
+        },
+        "class_name": "keras_cv.models>ViTBackbone",
+        "config": {
+            "stackwise_filters": [64, 128, 256, 512],
```

Comment:
This preset definitions still appear to be from the ResNet model. You can update once `ViTBackbone` is ready